packages/vuetify/src/components/VTooltip/VTooltip.tsx

```diff
@@ -9,6 +9,7 @@ import { makeVOverlayProps } from '@/components/VOverlay/VOverlay'
 import { forwardRefs } from '@/composables/forwardRefs'
 import { useProxiedModel } from '@/composables/proxiedModel'
 import { useScopeId } from '@/composables/scopeId'
+import { makeVariantProps, useVariant } from '@/composables/variant'
```

Comment:
I don't think variant makes sense because there are at least 3 that come to mind that would be a bad experience, **plain**, **outlined**, and **text**.

Reply:
That is true. I'll look at revising this tomorrow.